.github/workflows/test.yaml

```diff
@@ -164,27 +164,37 @@ jobs:
         - uses: crazy-max/ghaction-github-runtime@v2
         - name: Set env
           run: |
-            echo ERASER_IMG=eraser:test >> $GITHUB_ENV
-            echo MANAGER_IMG=manager:test >> $GITHUB_ENV
-            echo COLLECTOR_IMG=collector:test >> $GITHUB_ENV
-            echo SCANNER_IMG=scanner:test >> $GITHUB_ENV
+            echo ERASER_REPO=eraser >> $GITHUB_ENV
+            echo MANAGER_REPO=manager >> $GITHUB_ENV
+            echo COLLECTOR_REPO=collector >> $GITHUB_ENV
+            echo SCANNER_REPO=scanner >> $GITHUB_ENV
+
+            echo ERASER_TAG=test >> $GITHUB_ENV
+            echo MANAGER_TAG=test >> $GITHUB_ENV
+            echo COLLECTOR_TAG=test >> $GITHUB_ENV
+            echo SCANNER_TAG=test >> $GITHUB_ENV
         - name: Build eraser-manager
-          run: make docker-build-manager MANAGER_IMG=${MANAGER_IMG}
+          run: make docker-build-manager MANAGER_REPO=${MANAGER_REPO} MANAGER_TAG=test
```

Comment:
do we still need to pass the env vars to `make` if we are setting as env vars in the previous step?

Reply:
done

Reply:
i meant `MANAGER_REPO=${MANAGER_REPO} MANAGER_TAG=test` here, is this needed?

Reply:
I forgot to cherry-pick a commit from another branch. My bad.

Reply:
done